src/main/java/io/matrix/bot/constants/model/LogData.java

```diff
@@ -0,0 +1,38 @@
+package io.matrix.bot.constants.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+import static java.time.ZoneOffset.UTC;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Getter
+@Setter
+@NoArgsConstructor
+public class LogData {
+    private String source;
+    private String host;
+    private long timestamp;
+    private int level;
+    private String message;
+    private int code;
+    private String stack;
+
+    public LogData(final LocalDateTime dateTime,
+                   final String message,
+                   final int level,
+                   final String stack,
+                   final String host,
+                   final String source,
+                   final int code) {
+        this.source = source;
+        this.host = host;
+        this.timestamp = dateTime.toInstant(UTC).toEpochMilli();
```

Comment:
Why UTC here and in Logger.java local date? Shouldn't we pick one time zone and use it everywhere? I think it would make our debugging lives easier to align with one time zone.

Reply:
Because it's a conversion to unix timestamp witch is always in UTC.
So I convert local date time to instant UTC then get the epoch milli